server/views/components/button-button/index.njk

```diff
@@ -1,6 +1,6 @@
 <button class="btn {{modifierClasses}}" {{attributes}}>
   {% if icon %}
-    {% icon icon %}
+    {% icon icon, null, iconExtraClasses %}
```

Comment:
We could use [keyword arguments](https://mozilla.github.io/nunjucks/templating.html#keyword-arguments) (named parameters) here.

e.g.
```
{% icon icon, extraClasses=iconExtraClasses  %}
```